engine/session_live.js

```diff
@@ -254,7 +254,9 @@ class SessionLive {
         this.vodSegments[bw].push(segments[bw][segIdx]);
       }
       if (!segments[bw][segments[bw].length - 1].discontinuity) {
-        this.vodSegments[bw].push({ discontinuity: true });
+        this.vodSegments[bw].push({ discontinuity: true, cue: { in: true } });
+      } else {
+        segments[bw][segments[bw].length - 1]["cue"] = { in: true };
```

Comment:
to always add cue-in tag when switching from V2L->Live. 
Adding a cue-in tag this way is simpler than having to wait for fetching and parsing the first seg-Item from live source.